### PR TITLE
Update validation annotation used by the UI

### PR DESCRIPTION
**Pull request by bipuladh**
Updated the validation annotation(external.features.ocs.openshit.io/validation) to include new storage classes
These are required to perform validation in the UI.

### CODE REVIEW

File: tools/csv-merger/csv-merger.go [hunk 1]
```diff
@@ -619,7 +619,7 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	ocsCSV.Annotations["features.ocs.openshift.io/enabled"] = `["external"]`
 	// Used by UI to validate user uploaded metdata
 	// Metadata is used to connect to an external cluster
-	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["rook-ceph-retain-bucket"]}`
+	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", "rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["rook-ceph-retain-bucket", "ceph-rgw", "ceph-rbd", "cephfs"]}`
```
Comment (jarrpa) [inline tools/csv-merger/csv-merger.go:622]: Shouldn't you get rid of `rook-ceph-retain-bucket`?

Comment (bipuladh) [reply tools/csv-merger/csv-merger.go:622]: yes...

**Pull request merged**